cmake/modules/hunter_jobs_number.cmake

```diff
@@ -11,9 +11,14 @@ function(hunter_jobs_number jobs_options_varname toolchain_path)
   string(COMPARE EQUAL "${HUNTER_JOBS_NUMBER}" "" use_default_jobs)
   if(use_default_jobs)
     cmake_host_system_information(
-        RESULT jobs_number
+        RESULT l_cores
         QUERY NUMBER_OF_LOGICAL_CORES
     )
+	cmake_host_system_information(
+        RESULT p_cores
+        QUERY NUMBER_OF_PHYSICAL_CORES
+    )
+	math(EXPR jobs_number "${l_cores} * ${p_cores}")
```

Comment:
Please check that neither `l_cores` nor `p_cores` is 0 (substitute it with `1`). For example I got `l_cores=16`, `p_cores=0` on Windows + CMake 3.7.2 with [my processor](http://ark.intel.com/products/78931/Intel-Core-i7-4710MQ-Processor-6M-Cache-up-to-3_50-GHz) (which has 8 cores in fact).